test/definitions/mass_fraction_test.py

```diff
@@ -0,0 +1,37 @@
+from collections import namedtuple
+from pytest import approx, fixture, raises
+from symplyphysics import (
+    errors,
+    units,
+    Quantity,
+)
+from symplyphysics.definitions import mass_fraction as mass_fraction_law
+
+
+@fixture(name="test_args")
+def test_args_fixture():
+    m_i = Quantity(1 * units.kilogram)
+    m = Quantity(3 * units.kilogram)
+    Args = namedtuple("Args", ["m_i", "m"])
+    return Args(m_i=m_i, m=m)
+
+
+def test_basic_mass_fraction(test_args):
```

Comment:
Now, when you have `Fraction`, could you test it? What if we try to calculate the fraction of 3 kg in 1 kg total mass?